src/webdriver/session.cr

```diff
@@ -77,40 +77,42 @@ module Selenium
     end
 
     def execute(script, *args)
-      post("/execute", { script: script, args: args })
+      post("/execute", {script: script, args: args})
     end
 
     def execute_async(script, *args)
-      post("/execute", { script: script, args: args })
+      post("/execute", {script: script, args: args})
     end
 
     def frame(identifier)
-      post("/frame", { id: identifier })
+      post("/frame", {id: identifier})
     end
 
     def parent_frame
       post("/frame/parent")
     end
 
-    #def screenshot
-    #  # TODO: decode Base64
-    #  post("/screenshot")
-    #end
+    def screenshot(path = "./screenshot.png")
```

Comment:
or better with

```
      if File.writable?(path)
        File.write(path, decoded_image)
      else
        pp "Cannot write file to current directory"
        return false
      end
```

@ysbaddaden

Reply:
I'd say to let it raise, a developer should be capable to deal with the exception.

Reply:
you mean, let `File.write` raise his error & exception ? okay